pkg/controllermanager/controller/shoot/shoot_control_reconcile.go

```diff
@@ -143,6 +143,11 @@ func (c *defaultControl) reconcileShoot(o *operation.Operation, operationType ga
 			Fn:           flow.SimpleTaskFn(botanist.WaitUntilEtcdReady).SkipIf(o.Shoot.IsHibernated),
 			Dependencies: flow.NewTaskIDs(deployETCD),
 		})
+		_ = g.Add(flow.Task{
+			Name:         "Deleting orphan etcd main persistent volume due to recent migration",
+			Fn:           flow.TaskFn(botanist.DeleteOrphanEtcdMainPVC).SkipIf(o.Shoot.IsHibernated),
```

Comment:
Why is this skipped if the shoot is hibernated?

Reply:
Thank you for review. This was added before script based approach from George. I should have removed it post addition of script. Removed now.